third-party/arcanum-worlds/odyssey-of-the-dragonlords/items.xml

```diff
@@ -37,4 +35,707 @@
+			<p class="indent">The chariot has a flying speed of 90 ft. Flying is a free action, but the horses are difficult to control, and every round, the chariot rider must use a bonus action to control the horses. This requires a successful DC 15 Wisdom (Animal Handling) check. Failure means that the control is lost and the horses will move their full movement to climb straight up into the sky.</p>
+		</description>
+		<setters>
+			<set name="keywords">fly</set>
+			<set name="category">Wondrous Items</set>
+			<set name="cost" currency="gp">0</set>
+			<set name="weight" lb="0">0 lb.</set>
+			<set name="type">Wondrous Item</set>
+			<set name="attunement">true</set>
+			<set name="rarity">Legendary</set>
+		</setters>
+	</element>
+	<element name="Crown of Laurels" type="Magic Item" source="Odyssey of the Dragonlords" id="ID_AW_OOTD_MAGIC_ITEM_CROWN_OF_LAURELS">
+		<description>
+			<p>This crown of enchanted laurel leaves is customarily awarded to the winner of the Great Games. It was crafted by Vallus, the Queen of Mytros and the Goddess of Wisdom. When worn, the crown provides +2 to Wisdom saving throws.</p>
+		</description>
+		<setters>
+			<set name="keywords"></set>
+			<set name="category">Wondrous Items</set>
+			<set name="cost" currency="gp">0</set>
+			<set name="weight" lb="0">0 lb.</set>
+			<set name="slot">head</set>
+			<set name="type">Wondrous Item</set>
+			<set name="attunement">true</set>
+			<set name="rarity">Very Rare</set>
+		</setters>
+		<rules>
+			<stat name="wisdom:save:misc" value="2" />
+		</rules>
+	</element>
+	<element name="Crown of the Dragonlords" type="Magic Item" source="Odyssey of the Dragonlords" id="ID_AW_OOTD_MAGIC_ITEM_CROWN_OF_THE_DRAGONLORDS">
+		<description>
+			<p>If you are a spellcaster (bard, cleric, druid, paladin, ranger, sorcerer, warlock or wizard), this crown grants you the ability to prepare the following spells: <i>bond of the dragonlords</i> and <i>dirge of the dragonlords</i>. If you are not a spellcaster, then as long as you wear the crown, you can cast <i>bond of the dragonlords</i>. Once you cast it, you must finish a long rest before you can cast it again. The material components are still required for both spells.</p>
+		</description>
+		<setters>
+			<set name="keywords"></set>
+			<set name="category">Wondrous Items</set>
+			<set name="cost" currency="gp">0</set>
+			<set name="weight" lb="0">0 lb.</set>
+			<set name="slot">head</set>
+			<set name="type">Wondrous Item</set>
+			<set name="attunement">true</set>
+			<set name="rarity">Rare</set>
+		</setters>
+		<!--
+		<rules>
+			<grant type="Spell" id="ID_AW_OOTD_SPELL_BOND_OF_THE_DRAGONLORDS" requirements="type:spell" />
+			<grant type="Spell" id="ID_AW_OOTD_SPELL_DIRGE_OF_THE_DRAGONLORDS" requirements="type:spell" />
+		</rules>
+		-->
+	</element>
```

Comment:
the wording is a little off since not all spellcasteres prepare spells but you can add the spells to the list of all spellcasters like so
those who prepare can prepare them, those who choose can choose them (a non spellcaster can add the spell from the 'additional spells' equipment category if they want to add the card)
```
<spellcasting all="true" extend="true">
    <extend>ID_AW_OOTD_SPELL_BOND_OF_THE_DRAGONLORDS</extend>
    <extend>ID_AW_OOTD_SPELL_DIRGE_OF_THE_DRAGONLORDS</extend>
</spellcasting>
```